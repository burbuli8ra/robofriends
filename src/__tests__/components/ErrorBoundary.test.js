import React from 'react';
import { shallow } from 'enzyme';
import ErrorBoundary from '../../components/ErrorBoundary';

let wrapper;

beforeEach(() => {
  wrapper = shallow(<ErrorBoundary />);
});
describe('ErrorBoundary Component', () => {
  it('should render ErrorBoundary component', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('should render error component', () => {
    const mockErrorState = {
      hasError: true
    };

    expect(wrapper.instance().componentDidCatch()).toMatchSnapshot();
  });
});
