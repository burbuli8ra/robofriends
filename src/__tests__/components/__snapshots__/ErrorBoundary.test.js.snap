// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorBoundary Component should render ErrorBoundary component 1`] = `ShallowWrapper {}`;

exports[`ErrorBoundary Component should render error component 1`] = `undefined`;
